src/site/content/en/blog/houdini-how/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Cross-browser paint worklets and Houdini.how'
+subhead: CSS Houdini worklets and resources are just a few clicks away.
```

Comment:
High-level feedback: if you're targeting this at people who already know Houdini then these titles LGTM. But if you want this to make more people aware of the use cases that Houdini solves, I would update this title or subhead to mention the use cases. When I look at this title and subhead, the only information I get is 1) cross-browser and 2) existence of some site called houdini.how and 3) it's somehow related to CSS and 4) existence that Houdini supports some kind of thing called worklets

Reply:
The first 3 paragraphs in your intro seem to suggest main Houdini use cases, notice how none of that is hinted at here in the title or subhead

Reply:
"Focus on the tasks, not the features" is my guiding principle for titles and subheads (and general docs organization)

Reply:
The idea is cross-browser (new concept) and link to houdini.how (new launch)

We've written other articles about what Houdini is and how to use it. I reference them at the start of the article for context. Do you still think there should be more context in the titles?